api/src/pcapi/models/feature.py

```diff
@@ -47,7 +47,6 @@ class FeatureToggle(enum.Enum):
     ENABLE_AUTO_VALIDATION_FOR_EXTERNAL_BOOKING = (
         "Valide automatiquement après 48h les offres issues de l'api billeterie cinéma"
     )
-    ENABLE_BACKOFFICE_API = "Autorise l'accès à l'API backoffice"
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏Il manque la migration qui va avec 
Exemple: src/pcapi/alembic/versions/20221201T110438_51d8855b12f6_delete_collective_offer_choice_creation_ff.py